radar-auth/build.gradle

```diff
@@ -14,8 +14,8 @@ ext {
     junitVersion ='5.6.3'
 }
 
-sourceCompatibility = JavaVersion.VERSION_1_8
-targetCompatibility = JavaVersion.VERSION_1_8
+sourceCompatibility = JavaVersion.VERSION_11
```

Comment:
Wouldn't it make sense to use the same version across subprojects?

Reply:
The reason for the difference is that the libraries need some more compatibility, and application code can use the latest technologies.